run_scratchblocks4.js

```diff
@@ -1,22 +1,64 @@
-mw.hook('wikipage.content').add(function run_scratchblocks() {
-	var version = mw.config.get('wgScratchBlocks4BlockVersion');
+mw.hook("wikipage.content").add(function run_scratchblocks() {
+	var version = mw.config.get("wgScratchBlocks4BlockVersion");
```

Comment:
Unless you can give a technical reason for this pedantic change, please revert all of the unnecessary quote styling changes in this PR that inflate the diff.